infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/DirectKubernetesAPIAccessHelperTest.java

```diff
@@ -114,8 +114,8 @@ public void testSendsRequestHeaders() throws Exception {
             new ByteArrayInputStream("null".getBytes(StandardCharsets.UTF_8)));
 
     // then
-    assertEquals(requestCaptor.getValue().header("ducks"), "many");
-    assertEquals(requestCaptor.getValue().header("geese"), "volumes");
+    assertEquals(requestCaptor.getValue().header("ducks"), null);
```

Comment:
isn't this broken? Shouldn't there be real values set in the map above ?

Reply:
Nope in https://github.com/eclipse-che/che-server/pull/6 removed all headers except ```content/type```

Reply:
aha, I see. Maybe worth to update test name to sth like `testSendsOnlyContentTypeHeaders`?

Reply:
done